packages/react-scripts/package.json

```diff
@@ -28,66 +28,52 @@
   },
   "types": "./lib/react-app.d.ts",
   "dependencies": {
-    "@babel/core": "7.12.3",
-    "@babel/eslint-parser": "^7.13.14",
-    "@pmmmwh/react-refresh-webpack-plugin": "0.4.3",
+    "@babel/core": "7.14.2",
+    "@pmmmwh/react-refresh-webpack-plugin": "0.5.0-rc.0",
     "@svgr/webpack": "5.5.0",
-    "@typescript-eslint/eslint-plugin": "^4.5.0",
-    "@typescript-eslint/parser": "^4.5.0",
     "babel-jest": "^26.6.0",
-    "babel-loader": "^8.1.0",
+    "babel-loader": "8.2.2",
     "babel-plugin-named-asset-import": "^0.3.7",
     "babel-preset-react-app": "^10.0.0",
     "bfj": "^7.0.2",
-    "camelcase": "^6.1.0",
-    "case-sensitive-paths-webpack-plugin": "2.3.0",
-    "css-loader": "4.3.0",
-    "dotenv": "8.2.0",
+    "browserslist": "^4.16.6",
+    "camelcase": "^6.2.0",
+    "case-sensitive-paths-webpack-plugin": "2.4.0",
```

Comment:
This plugin reduce build time very, I recommend to use linter (`eslint` and ``) for this task https://github.com/import-js/eslint-plugin-import/blob/master/docs/rules/no-unresolved.md#casesensitive

Reply:
Thanks @alexander-akait good point - I've removed the plugin and added the import/no-unresolved _(caseSensitive check should be on pr. default - would be nice to limit the number of plugins and loaders in general)_

Reply:
@alexander-akait I've defered the suggestion to #11272 
_(I tried adding it quickly but ran into a wall regarding baseUrl configurations in tsconfig.json / jsconfig.json - hoped that the typescript resolver could solve the issue in general but tbh eslint shared preset configs/plugin setup is less than perfect, it quickly felt like a rabit hole - defering in order to get the v5 alpha out)_